### PR TITLE
Added files to feature branch for review

### PR DESCRIPTION
**Pull request by mnaibei**
Added todolist files and created project

### CODE REVIEW

**Review by abel-tefera — COMMENTED** [review]
Hello, Mucha



While you made great effort for ensuring JavaScript best practices in your project, there are still a few things left

File: .gitignore copy
Comment (abel-tefera) [inline .gitignore copy:1]: Kindly remove duplicate files from your commits

File: src/index.js
Comment (abel-tefera) [inline src/index.js:1]: Good usage of modularity

**Comment by mnaibei** [comment]
Thank you 😃

File: src/old-files/add.js
Comment (abel-tefera) [inline src/old-files/add.js:1]: Kindly remove the files in this folder from your commit history. Old pieces of code should be removed and possibly retrieved from the commit history

**Pull request merged**